client/my-sites/importer/importer-blogger.jsx

```diff
@@ -0,0 +1,47 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FileImporter from './file-importer';
+
+/**
+ * Module variables
+ */
+const importerData = {
+	title: 'Blogger.com',
+	icon: 'blogger',
+};
+
+class ImporterBlogger extends React.PureComponent {
+	static displayName = 'ImporterBlogger';
+
+	render() {
+		importerData.description = this.props.translate(
```

Comment:
We can improve on this pattern to avoid the slight misdirection that comes from having `importerData` up top and adding to it here...

See https://github.com/Automattic/wp-calypso/pull/23930#discussion_r179598308